lib/portlayer/storage/store_cache.go

```diff
@@ -108,9 +115,12 @@ func (c *NameLookupCache) GetImageStore(ctx context.Context, storeName string) (
 }
 
 func (c *NameLookupCache) CreateImageStore(ctx context.Context, storeName string) (*url.URL, error) {
+	defer trace.End(trace.Begin(storeName))
+
 	u, err := c.GetImageStore(ctx, storeName)
 	// we expect this not to exist.
 	if err == nil {
+		log.Debugf("Image store %s already exists", storeName)
```

Comment:
Same comment.  Caller should do the logging.  Update the error if it isn't sufficiently verbose then update the caller to print the log on err != nil